home-manager/firefox.nix

```diff
@@ -6,6 +6,7 @@ lib.mkMerge [
     # https://github.com/nix-community/home-manager/blob/release-24.05/modules/programs/firefox.nix
     programs.firefox = {
       enable = true;
+      package = pkgs.emptyDirectory;
```

Comment:
```

       error: attribute 'override' missing

       at /nix/store/q1ss75nhmjiydyqw8lbzwbx7cqk8z0qq-source/modules/programs/firefox.nix:208:7:

          207|     else if versionAtLeast config.home.stateVersion "19.09" then
          208|       package.override (old: {
             |       ^
          209|         cfg = old.cfg or { } // fcfg;
[cargo-make] ERROR - Error while executing command, exit code: 1
[cargo-make] WARN - Build Failed.
```

Reply:
Removed this section in recent version?

https://github.com/nix-community/home-manager/commit/792757f643cedc13f02098d8ed506d82e19ec1da#diff-e792456d03ba77b2e4c833fc50e36c102b4f589b58b56779b9d1420608ba32cfR2